.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
this has nothing to do with the task
Actually, I think, these vscode settings should be specific to personal development environment, so let's add .vscode folder to .gitignore